### PR TITLE
Resourses

### PR DESCRIPTION
**Pull request by Sara-cos**
## What does this PR do?
Add resources 

### Description
Added cloud computing free course and data science beginner course

### Why is this valuable (or not)?
The cloud computing course covers all the cloud computing topics and a beginner course in data science is really necessary

### How do we know it's really free?
Yes its free

### For book lists, is it a book? For course lists, is it a course? etc.
Its course.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -274,6 +275,7 @@
 * [Data Analysis with R](https://www.udacity.com/course/data-analysis-with-r--ud651) - Facebook (Udacity)
 * [Data Cleaning by Rachael Tatman at Kaggle](https://www.kaggle.com/learn/data-cleaning)
 * [Data Mining with Weka MOOC](https://www.cs.waikato.ac.nz/ml/weka/mooc/dataminingwithweka/)
+* [Data science for beginners](https://github.com/microsoft/Data-Science-For-Beginners) - Microsoft
```
Comment (davorpa) [inline courses/free-courses-en.md:278]: Would be better the front-matter?
> https://microsoft.github.io/Data-Science-For-Beginners

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -246,6 +246,7 @@
 
 ### Cloud Computing
 
+* [Cloud Computing Full Course In 11 Hours](https://www.youtube.com/watch?v=2LaAJq1lB1Q) - Edureka
```
Comment (davorpa) [inline courses/free-courses-en.md:249]: - Completing title
- Fix typo in platform/instructor? `Edureka` vs `Edureka!`

```suggestion
* [Cloud Computing Full Course In 11 Hours \| Cloud Computing Tutorial For Beginners](https://www.youtube.com/watch?v=2LaAJq1lB1Q) - Edureka!
```

What do we think about?

File: courses/free-courses-en.md (hunk 1 above)
Comment (Sara-cos) [reply courses/free-courses-en.md:278]: Yes that would be more learner friendly interface.

File: courses/free-courses-en.md (hunk 2 above)
Comment (davorpa) [reply courses/free-courses-en.md:249]: if you are agree, apply both suggestion

**Comment by eshellman** [comment]
I thing @Sara-cos misunderstood the title suggestion. We try to use the title from the resource, which would be  `Cloud Computing Full Course In 11 Hours` or more completely `Cloud Computing Full Course In 11 Hours \| Cloud Computing Tutorial For Beginners`. The second part is a subtitle, so optional.

One reason we do this so we can more easily tell if we already have a resource.

File: courses/free-courses-en.md (hunk 2 above)
Comment (davorpa) [reply courses/free-courses-en.md]: Not resolved yet https://github.com/EbookFoundation/free-programming-books/pull/6581#discussion_r740621040 as Eric mention on https://github.com/EbookFoundation/free-programming-books/pull/6581#issuecomment-959191238

File: courses/free-courses-en.md (hunk 1 above)
Comment (davorpa) [reply courses/free-courses-en.md]: Resolved at https://github.com/EbookFoundation/free-programming-books/commit/1ba26e533ca56800b4d2c91674b15fd67f23792d#diff-b0eb8fcca67d6470c73302a9fa829288753e4a1a31ef6815df89165d8676dcd2R278

File: courses/free-courses-en.md (hunk 2 above)
Comment (davorpa) [reply courses/free-courses-en.md]: Definitely resolved at https://github.com/EbookFoundation/free-programming-books/pull/6581/commits/2923ed28272358ef9095b40de50a7d10e4b77d79

**Review by davorpa — APPROVED** [review]

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**